src/linagora.esn.unifiedinbox/app/components/composer/composer.controller.js

```diff
@@ -168,6 +168,15 @@ angular.module('linagora.esn.unifiedinbox')
       _.forEach(self.message.attachments, _cancelAttachment);
 
       self.draft.destroy(options).then(self.onDiscard, self.onShow);
+
+      if (typeof self.onDiscarding !== 'function') return;
```

Comment:
Missing test for this?

Reply:
It's just a simple guard check so it doesn't really make sense to write a test for it.

Reply:
Ok